migrator.go

```diff
@@ -45,25 +44,15 @@ func (m spannerMigrator) AutoMigrate(values ...interface{}) error {
 			return err
 		}
 	}
-	inputs := make([]reflect.Value, len(values))
-	for i, _ := range values {
-		inputs[i] = reflect.ValueOf(values[i])
-	}
-	result := reflect.ValueOf(&m.Migrator).MethodByName("AutoMigrate").Call(inputs)
-	if len(result) != 1 {
-		return fmt.Errorf("unexpected return value length: %d", len(result))
-	}
-	if err, ok := result[0].Interface().(error); ok {
-		return err
-	}
-	if result[0].IsNil() {
+	err := m.Migrator.AutoMigrate(values...)
```

Comment:
I'm surprised that this works. I tried this multiple times as well, as it is by far the most logical way, but I was never able to get it work (and I don't remember why....)